src/js/services/apiservice.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
remove all this whitespace

Reply:
line 1-4

Reply:
I dont wanna comment on every line, but have a look at your PR again and remove it from everywhere not only these lines

Reply:
Fixed Thank you for your feedback